src/pages/contact.js

```diff
@@ -56,14 +72,18 @@ const Contact = () => {
             Lorem ipsum dolor sit amet, consectetur adipisicing elit.
             Itaque, fuga!
           </p>
-          <Form onSubmit={onSubmit}>
+          <form 
```

Comment:
Why don't you use a component we already have?

Reply:
You can change it, it was allowing redirections, from action,  which I could  checked why but didn't. Could you change if you get some time